controller/SynchronisationApi.php

```diff
@@ -206,10 +207,28 @@ public function confirmSyncFinished()
             $eventManager->trigger(new SyncFinishedEvent($syncParams, $report));
 
             $this->returnJson(['message' => 'Confirmation received.']);
-        } catch (\Exception $e) {
-            $eventManager->trigger(new SyncFinishedEvent($syncParams, $report));
-            $this->returnFailure($e);
-        }
+        } catch (\Exception $e) {}
```

Comment:
could we place here anything? at least Logger::d?

Reply:
Done